cypress/e2e/tables.cy.js

```diff
@@ -0,0 +1,14 @@
+import {User} from "@nextcloud/cypress"
+describe('The Home Page', () => {
+	it('successfully loads', () => {
+		const user = new User('admin', 'admin')
+		cy.login(user)
+      	cy.visit('apps/tables') // change URL to match your dev URL
+	  	cy.contains('.app-menu-entry--label', 'Tables').click()
+		cy.contains('button','Create new table').click()
+		cy.get('input[name="template"][value="todo"]').click({force: true})
+		cy.get('.modal__content input[type="text"]').type('to do list')
+		cy.contains('button','Create table').click()
+		cy.contains('button','Create row').should('be.visible')
```

Comment:
Would be nice to additionally check the the title is properly displayed and the newly created table is actually visible